backend/uclapi/timetable/management/commands/update_gencache.py

```diff
@@ -358,5 +377,17 @@ def handle(self, *args, **options):
             except Exception as teams_error:
                 print(f"Failed to send message to Microsoft Teams. "
                       f"Reason: {repr(teams_error)}")
+
+            incident_name = get_incident_name("Gencache")
+            if incident_name is not None:
```

Comment:
```suggestion
            if incident_name:
```
This should work not matter if the default values returned by `get_incident_name()` is `None` or `""`.

Reply:
I opted to go for the more verbose method since then the programmer knows exactly what we are expecting back in the default case.